src/components/HomePage/actions.ts

```diff
@@ -0,0 +1,14 @@
+import { INCREMENT, DECREMENT, ADD_COUNTER } from './actionTypes';
+import { IAction } from './types'
+
+export const addCounterAction = ():IAction => {
```

Comment:
This should be in containers folder.

src/containers/HomePage/actions.ts
```ts
export const increment: ActionCreator.IncreaseValue = (id) => ({
  type: ActionType.INCREMENT,
  payload: {
    id,
  },
});
```

Honestly, `increment` is not very suitable here, because it is a noun. Better to have name saying about action (call to action), starting from a verb, e.g., `increaseValue`.